packages/liquid-html-parser/src/stage-1-cst.ts

```diff
@@ -369,14 +370,20 @@ export interface ConcreteLiquidFilter extends ConcreteBasicNode<ConcreteNodeType
   args: ConcreteLiquidArgument[];
 }
 
-export type ConcreteLiquidArgument = ConcreteLiquidExpression | ConcreteLiquidNamedArgument;
+export type ConcreteLiquidArgument = ConcreteLiquidExpression | ConcreteLiquidNamedArgument | ConcreteLiquidContentForNamedArgument;
 
 export interface ConcreteLiquidNamedArgument
   extends ConcreteBasicNode<ConcreteNodeTypes.NamedArgument> {
   name: string;
   value: ConcreteLiquidExpression;
 }
 
+export interface ConcreteLiquidContentForNamedArgument
+  extends ConcreteBasicNode<ConcreteNodeTypes.ContentForNamedArgument> {
+  name: string;
+  value: ConcreteLiquidExpression;
+}
```

Comment:
Does this really need a new concrete node type? I see we fold it back into NamedArgument in stage 2. Why not fold it in the contentForTagArgument mapping instead?